setup.py

```diff
@@ -22,12 +21,14 @@
     project_urls={"Documentation": "https://docs.meilisearch.com/",},
     keywords="search python meilisearch",
     platform="any",
-    classifiers=[
+classifiers=[
```

Comment:
Spacing is off here. This is probably my fault when I put in the suggestion.

```suggestion
    classifiers=[
````